src/entities/activity/lib/types/item.ts

```diff
@@ -22,3 +39,52 @@ export type ItemCardButtonsConfig = {
   isSkippable?: boolean
   isNextDisable?: boolean
 }
+
+export type BaseItem<Config, ResponseValues> = {
+  id: string
+  name: string
+  question: string
+  responseType: ActivityItemType
+  responseValues: ResponseValues | null
+  order: number
+  config: Config
+  answer: string[]
+}
+
+export type TextItemConfig = {
+  maxResponseLength: number // default 300
+  correctAnswerRequired: boolean // default false
+  correctAnswer: string // default ""
+  numericalResponseRequired: boolean // default false
+  responseDataIdentifier: string // default ""
+  responseRequired: boolean // default false
+  removeBackButton: boolean
+  skippableItem: boolean
+}
+
+export type CheckboxConfig = {
+  removeBackButton: boolean
+  skippableItem: boolean
+  randomizeOptions: boolean
+  timer: number | null
+  addScores: boolean
+  setAlerts: boolean
+  addTooltip: boolean
+  setPalette: boolean
+  additionalResponseOption: {
+    textInputOption: boolean
+    textInputRequired: boolean
+  }
+}
```

Comment:
I'd propose refactoring some of the properties into separate types:

```
type ButtonsConfig = {
  removeBackButton: boolean;
  skippableItem: boolean;
};

type TimerConfig = {
  timer: number | null;
};

type AdditionalResponseConfig = {
  additionalResponseOption: {
    textInputOption: boolean;
    textInputRequired: boolean;
  };
};
```
They are very common among the other item DTOs. So you can use them to compose different types without writing too much boilerplate in the future